### PR TITLE
Updated the workflow guide to remove the deprecated method.

### PR DESCRIPTION
**Pull request by sagarkapare**
Removed the deprecated `getMapReduceCounters` method and added unit test case.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -112,9 +112,7 @@ and overrides the ``configure`` method to define all of the application componen
       addMapReduce(new PurchaseCounterByProduct());
       addWorkflow(new PurchaseWorkflow());
 
-      scheduleWorkflow(Schedules.builder("HourlySchedule")
-                         .setDescription("Schedule execution every 1 hour")
-                         .createTimeSchedule("0 * * * *"),
+      scheduleWorkflow(Schedules.createTimeSchedule("HourlySchedule", "Schedule execution every 1 hour", "0 * * * *"),
```
Comment (bdmogal) [inline README.rst]: This changes it back to a deprecated method. Should revert this change

File: src/main/java/co/cask/cdap/guides/workflow/PurchaseWorkflowApp.java [hunk 2]
```diff
@@ -21,9 +21,7 @@ public void configure() {
     addMapReduce(new PurchaseCounterByProduct());
     addWorkflow(new PurchaseWorkflow());
 
-    scheduleWorkflow(Schedules.builder("HourlySchedule")
-                       .setDescription("Schedule execution every 1 hour")
-                       .createTimeSchedule("0 * * * *"),
+    scheduleWorkflow(Schedules.createTimeSchedule("HourlySchedule", "Schedule execution every 1 hour", "0 * * * *"),
```
Comment (bdmogal) [inline src/main/java/co/cask/cdap/guides/workflow/PurchaseWorkflowApp.java]: ditto

**Comment by bdmogal** [comment]
couple of comments about usage of deprecated methods. Otherwise LGTM

**Comment by sagarkapare** [comment]
Reverted the changes for schedule creation.

**Comment by bdmogal** [comment]
:+1:
